hoomd/mpcd/SphereGeometry.h

```diff
@@ -85,25 +107,31 @@ class __attribute__((visibility("default"))) SphereGeometry
              * The perpendicular and parallel components of the velocity are:
              * v_perp = (v.n)n = (v.r/R^2)r
              * v_para = v-v_perp
+	     * V_vec is vector component of V(interface velocity)
              */
+	    const Scalar3 V_vec = m_V*pos/(m_R-m_V*dt);
+	    const Scalar3 vperp = (dot(vel,pos)/m_R2)*pos;
```

Comment:
I think this is still not quite right when you are taking the dot product for `vperp`, since the radius of the droplet is not necessarily `m_R`. Suggest
```suggestion
	    const Scalar R = m_R - m_V*dt;
	    const Scalar3 V_vec = m_V*pos/R;
	    const Scalar3 vperp = dot(vel, pos)*pos/(R*R);
```